actions/create_xcconfig.rb

```diff
@@ -1,20 +1,20 @@
 module Fastlane
   module Actions
     class CreateXcconfigAction < Action
-      def self.run(params)
+      def self.run(params) # rubocop:disable Metrics/AbcSize
         filename = params[:filename]
         project = Xcodeproj::Project.open(ENV['FL_PROJECT_SIGNING_PROJECT_PATH'])
         lines = []
         project.targets.each do |target|
-          profile = self.get_profile_for_target(target)
-          if !profile.nil?
+          profile = get_profile_for_target(target)
+          if !profile.nil? # rubocop:disable Style/NegatedIf, Style/IfUnlessModifier
             lines.push("#{target.name}_PROFILE_SPECIFIER=#{profile}")
           end
         end
-        lines.push("PROVISIONING_PROFILE_SPECIFIER=$($(TARGET_NAME)_PROFILE_SPECIFIER)")
+        lines.push('PROVISIONING_PROFILE_SPECIFIER=$($(TARGET_NAME)_PROFILE_SPECIFIER)')
         begin
-          File.write(filename, lines.join("\n"))
-        rescue => exception
+          File.write(filename, lines.join('\n'))
+        rescue => exception # rubocop:disable Style/RescueStandardError
```

Comment:
Ruby is expecting this: 
```ruby
rescue StandardError => exception
```
As it likes it when you specify the types of errors that are getting rescued

Reply:
This isn't a vital one though

Reply:
ahh okay!